src/aem/azext_aem/__init__.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+
+import azext_aem._help  # pylint: disable=unused-import
+
+
+class AEMCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        aem_custom = CliCommandType(
+            operations_tmpl='azext_aem.custom#{}')
+        super(AEMCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                custom_command_type=aem_custom)
+
+    def load_command_table(self, _):
+        with self.command_group('vm aem') as g:
+            g.custom_command('set', 'set_aem')
+            g.custom_command('delete', 'delete_aem')
+            g.custom_command('verify', 'verify_aem')
+
+        return self.command_table
+
+    def load_arguments(self, _):
+        from knack.arguments import CLIArgumentType
+        from azure.cli.core.commands.parameters import get_resource_name_completion_list
+        name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
+        existing_vm_name = CLIArgumentType(overrides=name_arg_type,
```

Comment:
Would be nice to pick up the [common param](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py#L122) from azure-cli-vm, but can't get it work so far.